spec/support/sdam_monitoring/replica_set_with_removal.yml

```diff
@@ -0,0 +1,64 @@
+description: "Monitoring a replica set with non member"
+uri: "mongodb://a,b/?replicaSet=rs"
+phases:
+  -
+    responses:
+      "a:27017":
+        ok: 1
+        ismaster: true
+        setName: "rs"
+        setVersion: 1.0
+        hosts: [ "a:27017" ]
+        minWireVersion: 0
+        maxWireVersion: 4
+      "b:27017":
+        ok: 1
+        ismaster: true
+    outcome:
+      events:
+        -
+          topology_opening_event:
+            topologyId: "42"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "a:27017"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "b:27017"
+        -
+          server_description_changed_event:
+            topologyId: "42"
+            address: "a:27017"
+            previousDescription:
+              address: "a:27017"
+              arbiters: []
+              hosts: []
+              passives: []
+              primary: null
+              setName: null
+              type: "Unknown"
+            newDescription:
+              address: "a:27017"
+              arbiters: []
+              hosts: [ "a:27017" ]
+              passives: []
+              primary: true
+              setName: "rs"
+              type: "RSPrimary"
+        -
+          server_closed_event:
+            topologyId: "42"
+            address: "b:27017"
+        -
+          topology_description_changed_event:
+            topologyId: "42"
+            previousDescription:
+              topologyType: "Unknown"
+              setName: null
+              servers: [ "a:27017" ]
```

Comment:
This should have "b:27017" as well.

It should also be an array of ServerDescription instead of array of string